cli/transactions/cli.py

```diff
@@ -0,0 +1,107 @@
+"""
+Provide implementation of the command line interface's transaction info commands.
+"""
+import asyncio
+import click
+from aiohttp_json_rpc import RpcGenericServerDefinedError
+
+from cli.utils import (
+    validate_ids,
+    validate_limit,
+    validate_head_sign,
+    validate_id,
+    get_network,
+    dict_to_pretty_json,
+)
+
+from cli.transactions.service import Transactions
+from cli.transactions.help import (
+    LIST_TRANSACTIONS_HEAD_HELP_MESSAGE,
+    LIST_TRANSACTIONS_START_HELP_MESSAGE,
+    LIST_TRANSACTIONS_LIMIT_HELP_MESSAGE,
+    LIST_TRANSACTIONS_REVERSE_HELP_MESSAGE,
+    LIST_TRANSACTIONS_FAMILY_NAME_HELP_MESSAGE,
+    LIST_TRANSACTIONS_IDS_HELP_MESSAGE,
+    TRANSACTION_ID_HELP_MESSAGE,
+)
+from cli.constants import (
+    HEADER_SIGNATURE_REGEXP,
+    NODE_URL_ARGUMENT_HELP_MESSAGE,
+)
+
+loop = asyncio.get_event_loop()
+
+
+@click.group('transactions', chain=True)
+def transaction_command():
+    """
+    Provide commands for working with transactions.
+    """
+    pass
+
+
+@click.option('--ids', type=str, help=LIST_TRANSACTIONS_IDS_HELP_MESSAGE)
```

Comment:
Add `required=False` everywhere, to node-url too.

Reply:
@anastasiia-bilova, done.